src/nspawn/nspawn.c

```diff
@@ -1219,7 +1219,13 @@ static int setup_timezone(const char *dest) {
         /* Fix the timezone, if possible */
         r = readlink_malloc("/etc/localtime", &p);
         if (r < 0) {
-                log_warning("/etc/localtime is not a symlink, not updating container timezone.");
+                log_warning("host's /etc/localtime is not a symlink, not updating container timezone.");
```

Comment:
@martinpitt Made change you suggested, shorter message, and no longer breaks build.